src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -186,6 +190,9 @@ public function shutdown()
         }
 
         $groups = ['unsilenced', 'remaining self', 'remaining direct', 'remaining indirect', 'legacy', 'other'];
+        if ($configuration->scream()) {
+            $groups[] = 'muted';
+        }
```

Comment:
@nicolas-grekas tell me if you think this is not worth it and I will remove it, since it was not requested.

Reply:
Is there a use case? I'm not sure: deprecations by phpunit are not really actionable. Bumping phpunit version is decoupled task.

Reply:
I didn't think of a particular use case, and wrote that when I wasn't aware this was to be done only when the caller is phpunit, so I will remove it.